frontend/src/components/Layout/Header/SearchCourses/SearchResultItemContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+import { searchClientCourses } from 'api/myCourses';
+import { Course } from 'types/course';
+import { ClientCourse } from 'types/clientCourse';
+
+import SearchResultItem from './SearchResultItem';
+
+interface CoursesFound {
+  course: Course;
+  addDivider: boolean;
+}
+
+const SearchResultItemContainer: React.FC<CoursesFound> = ({ course, addDivider }) => {
+  const [foundInMyCourses, setFoundInMyCourses] = useState<string | undefined>(undefined);
+
+  const findCourse = (courses: Array<ClientCourse>, title: string): void => {
+    for (let clientCourse = 0; clientCourse < courses.length; clientCourse += 1) {
```

Comment:
```suggestion
    for (let clientCourse = 0; clientCourse < courses.length; clientCourse ++) {
```
just to avoid magic numbers

Reply:
Unary operators aren't allowed in our app